### ISSUE TITLE
Events don't fire

### ISSUE THREAD

**Issue by pandermatt** [miguelgrinberg/turbo-flask#25]
Hello

```
$(document).on('turbo:render', function( e ) {
        console.log('hey')
});
```

This will result in no output, but the content of the website changes. Are there any other events to use upon a replace?

Thanks

**Comment by pandermatt** [comment]
Sadly, this also emits no event. I also tried other events, but none are triggered

**Comment by miguelgrinberg** [comment]
I really have no control over this, it is all handled by the turbo library.

I did a quick test to confirm that events are dispatched, and they are. I added the following to my "todos" example: 

```html
    <script>
      document.addEventListener('turbo:submit-start', () => console.log('submit-start'));
      document.addEventListener('turbo:submit-end', () => console.log('submit-end'));
    </script>
```

These work fine, so it is not a dispatching problem. My guess is that the event doesn't fire because it isn't supposed to fire in the situation you are in. I suggest asking on Stack Overflow or a hotwired forum.
